src/authZConfig.ts

```diff
@@ -22,7 +22,7 @@ export const scopeRule: ScopeRule = {
 };

```

Comment:
We might want to make the same change to the system.write array that was made for user.write, the addition of 'delete' to the list.